lib/open_project/journal_formatter/custom_field/plain.rb

```diff
@@ -93,10 +93,12 @@ def find_user_value(value, _custom_field)
   def find_item_value(value, _custom_field)
     ids = value.split(",").map(&:to_i)
 
-    CustomField::Hierarchy::Item.where(id: ids).map do |item|
-      next I18n.t(:label_deleted_custom_option) unless ids.include?(item.id)
+    items = CustomField::Hierarchy::Item.where(id: ids).index_by(&:id)
 
-      item.ancestry_path
+    ids.map do |id|
+      next I18n.t(:label_deleted_custom_option) unless items[id]
```

Comment:
Can we have a new key? as `option` is not the correct domain term for this. 

"(deleted item)"